From 7fc087dfd985e7e6e12659637b268129f60605de Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Wed, 30 May 2012 16:01:53 +0200
Subject: [PATCH] Revert "Allow FANCYTTY to be sourced from /etc/default/rcS."

This reverts commit ce9dcceef7745798cf92b2aac03164818b79ce63.

This avoids one ugly `set +u` in the $() subshell and avoids the hijack of /etc/default/rcS.

Reopens: #673207

Closes: #675162
Reported-by: Kamen Naydenov <pau4o@kamennn.eu>
Closes: #675197
Reported-by: Steve Cotton <steve@s.cotton.clara.co.uk>
Thanks-To: Julien Cristau <jcristau@debian.org>
---
 debian/lsb-base.NEWS          |    2 +-
 debian/lsb-base.README.Debian |    3 ++-
 init-functions                |    2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/debian/lsb-base.NEWS b/debian/lsb-base.NEWS
index 40c30fe..476ddc6 100644
--- a/debian/lsb-base.NEWS
+++ b/debian/lsb-base.NEWS
@@ -12,6 +12,6 @@ lsb (4.1+Debian1) unstable; urgency=low
   or a red [FAIL] depending on the daemon exit status.
 
   The "Fancy output" can be disabled by setting the FANCYTTY variable to 0
-  in the /etc/default/rcS configuration file.
+  in the /etc/lsb-base-logging.sh configuration file.
 
  -- Didier Raboud <odyx@debian.org>  Thu, 19 Apr 2012 11:25:01 +0200
diff --git a/debian/lsb-base.README.Debian b/debian/lsb-base.README.Debian
index 2a3637c..3f86d93 100644
--- a/debian/lsb-base.README.Debian
+++ b/debian/lsb-base.README.Debian
@@ -193,7 +193,8 @@ Bourne-style shell permitted by Debian policy (i.e. not just bash).
 
   - FANCY OUTPUT
 
-"Fancy output" can be overridden by setting FANCYTTY=0 in /etc/default/rcS.
+"Fancy output" can be overridden by setting FANCYTTY=0 in
+/etc/lsb-base-logging.sh .
 
   - OUTPUT FUNCTIONS
 
diff --git a/init-functions b/init-functions
index d8a7e98..0bfea9e 100644
--- a/init-functions
+++ b/init-functions
@@ -425,5 +425,5 @@ for hook in $(run-parts --lsbsysinit --list /lib/lsb/init-functions.d 2>/dev/nul
     [ -r $hook ] && . $hook || true
 done
 
-FANCYTTY=$([ -e /etc/default/rcS ] && . /etc/default/rcS && echo $FANCYTTY)
+FANCYTTY=
 [ -e /etc/lsb-base-logging.sh ] && . /etc/lsb-base-logging.sh || true
-- 
1.7.2.5

